src/sentry/backup/exports.py

```diff
@@ -68,6 +136,7 @@ def _export(
         printer.echo(errText, err=True)
         raise RuntimeError(errText)
 
+    cache = checkpointer if checkpointer is not None else NoopExportCheckpointer(None, printer)
```

Comment:
So essentially, NoopExportCheckpointer is being used in the short term and this PR is establishing the framework for using ExportCheckpointer in the future?